.travis.yml

```diff
@@ -2,19 +2,15 @@
 sudo: false
 language: ruby
 cache: bundler
-env:
+env: 
 rvm:
-  - jruby-1.7.25
+- jruby-1.7.25
```

Comment:
should this be jruby 9k ?

Also if make this change, do we need 3 builds ?  It seems that the default (env blank) would end up being the same as LOGSTASH_BRANCH=master.